### PR TITLE
[integrations][Cloudflare Logpush] - Added local SQS queues per data stream, that will override the global queue if provided. 

### LINKED ISSUE #5694: Enhance Cloudflare Logpush AWS Data Stream UI
The AWS input for CL has an issue where we currently allow only one SQS que per instance for all the data streams. This has led to scaling and throughput issues for users as well made the integration a bit convoluted to use effectively. 

A related public issue : https://github.com/elastic/integrations/issues/5526

Enhance the CL AWS UI to allow optional queues at every data stream level, which can be overridden by a global queue

### PR DESCRIPTION
**Pull request by ShourieG**
## Type of change
- Enhancement

## What does this PR do?
This PR adds support for optional local SQS queue urls in each data stream. These queue urls will override the global queue if provided. This will improve scalability for users as previously they would often at times be limited/bottlenecked by one queue. 

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Related issues

- Closes #5694

### CODE REVIEW

File: packages/cloudflare_logpush/data_stream/dns/manifest.yml [hunk 1]
```diff
@@ -60,6 +60,13 @@ streams:
     description: Collect DNS logs from Cloudflare.
     template_path: aws-s3.yml.hbs
     vars:
+      - name: queue_url_dns
+        type: text
+        title: "[DNS][SQS] Queue URL"
+        multi: false
+        required: false
+        show_user: true
+        description: "URL of the AWS SQS queue that messages will be received from. \nIt is a required parameter for collecting logs via the AWS SQS.\nThis is a dns data stream specific queue URL. This will override the global queue URL if provided."
```
Comment (kcreddy) [inline packages/cloudflare_logpush/data_stream/dns/manifest.yml:69]: I am not sure if the `\n` changes to the newline. Could you please confirm and resolve this if it does change? Maybe remove it if it does nothing. Same for the rest of the datastreams

File: packages/cloudflare_logpush/data_stream/http_request/manifest.yml [hunk 2]
```diff
@@ -60,6 +60,13 @@ streams:
     description: Collect HTTP Request logs from Cloudflare.
     template_path: aws-s3.yml.hbs
     vars:
+      - name: queue_url_http_request
+        type: text
+        title: "[Http Request][SQS] Queue URL"
```
Comment (kcreddy) [inline packages/cloudflare_logpush/data_stream/http_request/manifest.yml:65]:
```suggestion
        title: "[HTTP Request][SQS] Queue URL"
```

File: packages/cloudflare_logpush/data_stream/dns/manifest.yml (hunk 1 above)
Comment (ShourieG) [reply packages/cloudflare_logpush/data_stream/dns/manifest.yml:69]: yea it does .. I first wrote it without \n but the elastic-package build and lint commands converted it to this .

File: packages/cloudflare_logpush/data_stream/audit/manifest.yml [hunk 3]
```diff
@@ -60,6 +60,13 @@ streams:
     description: Collect Audit logs from Cloudflare.
     template_path: aws-s3.yml.hbs
     vars:
+      - name: queue_url_audit
+        type: text
+        title: "[Audit][SQS] Queue URL"
+        multi: false
+        required: false
+        show_user: true
+        description: "URL of the AWS SQS queue that messages will be received from. \nIt is a required parameter for collecting logs via the AWS SQS.\nThis is an audit data stream specific queue URL. This will override the global queue URL if provided."
```
[the remark this answers is not in the document]
Comment (ShourieG) [reply packages/cloudflare_logpush/data_stream/audit/manifest.yml:69]: the desc is It's  required if we are collecting logs via SQS ... if not then it's not required ..but will add to the description to make it more clear

**Comment by ShourieG** [comment]
@kcreddy I've made the suggested changes.

**Review by kcreddy — APPROVED** [review]

**Pull request merged**